### PR TITLE
Odd Test

### PR DESCRIPTION
**Pull request by csbarrett1**

### CODE REVIEW

**Comment by abidina** [comment]
:+1:

**Comment by csbarrett1** [comment]
@katerebekah

File: Mathmagician.Tests/OddNumTests.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        public void OddSequenceOfNumbers()
+        {
+            //Arrange
+            OddNum my_odd = new OddNum();
+            //Act
+            List<int> NumberSequence = my_odd.Sequence(6);
+            List<int> ExpectedSequence = new List<int> { 1, 3, 5, 7, 9, 11 };
+            //Assert
+            Assert.AreEqual(ExpectedSequence.Count, NumberSequence.Count);
+        }
```
Comment (katerebekah) [inline Mathmagician.Tests/OddNumTests.cs:57]: Add a line here with a CollectionAssert.AreEqual to test if the lists/sequences are equal.

**Comment by katerebekah** [comment]
You'll need to make the Odd class's actual implementation at some point, but the tests look great with the addition I mentioned.

**Comment by katerebekah** [comment]
:+1:

**Pull request merged**